api/resolvers/item.js

```diff
@@ -1150,15 +1151,9 @@ export default {
         poll.meVoted = false
       }
 
-      poll.options = options
+      poll.randPollOptions = item?.randPollOptions
+      poll.options = poll.randPollOptions ? shuffleArray(options) : options
       poll.count = options.reduce((t, o) => t + o.count, 0)
-      const pollSettings = await models.item.findFirst({
-        where: {
-          id: item.id
-        },
-        select: { randPollOptions: true }
-      })
-      poll.randPollOptions = pollSettings?.randPollOptions || false
```

Comment:
The argument `item` had all the `item` members including `randPollOptions`.

I also decided it would be better to randomize on the server so that we don't have to rely on the client doing it. It's not necessarily better, but in general I prefer the server do work so clients are "thinner."